number_of_fpgas: 5
number_of_motors: 34
controller: 2
max_force: 10000
min_force: 0
qp_print_level: 0
qp_working_sets: 1000
log: 0
end_effectors:
  ["top"]

top:
  ForceControl:
    Kp: 1000
    Kd: 100
  TorqueControl:
    Kp: 1000
    Kd: 10
  PositionControl:
    Kp: 1000
    Kd: 0
  target_frame: ""
  target_offset: [0,0,0]
  target_rotation: [0,0,0,1]
  target_pos: [0,0,0]
  target_vel: [0,0,0]
  kinematic_chain: ["base","top"]
  joint_angle_mask: 0x0
  fpga_id: 5
  number_of_motors: 8